scene/3d/light_3d.cpp

```diff
@@ -165,6 +165,16 @@ AABB Light3D::get_aabb() const {
 	return AABB();
 }
 
+PackedStringArray Light3D::get_configuration_warnings() const {
+	PackedStringArray warnings = Node::get_configuration_warnings();
+
+	if (get_scale() != Vector3(1, 1, 1)) {
```

Comment:
Might need to use approx equal to make sure we don't get spurious warnings on floating point precision issues? (Should hopefully not happen with unchanged default scale but who knows.)